<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Play!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leaderboards_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Leaderboards</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
